assembly_pipeline_bwa/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Docker container for BWA
+
+FROM ubuntu:20.04
```

Comment:
This looks _very_ similar to https://github.com/EBI-Metagenomics/containers/blob/master/bwamem2/Dockerfile

Reply:
Maybe we should merge them

Reply:
It is indeed identical! I just needed the script to be added to the container. I can add it to the other one - but we were unsure of tracking back where the scripts come from. Option to also put the scripts in separate directories within the container?